tests/macros/InterfaceOverview.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Test the API Interface Overview page content macro
+ *
+ * @prettier
+ */
+
+// Get necessary modules
+const fs = require('fs');
+const path = require('path');
+
+const { assert, describeMacro, beforeEachMacro, itMacro } = require('./utils');
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+
+const INTERFACE_DATA = `[${JSON.stringify({
+    XMLHttpRequestEventTarget: {
+        inh: 'EventTarget',
+        impl: []
+    }
+})}]`;
+
+const DATA_DIR = path.resolve(__dirname, 'fixtures', 'interface-overview');
+const DATA = ((...dataNames) => {
+    /** @type {Record<string, object>} */
+    let result = {};
+    for (let i = 0; i < dataNames.length; i++) {
+        const name = dataNames[i];
+        result[name] = JSON.parse(
+            fs.readFileSync(path.resolve(DATA_DIR, `${name}.json`))
+        );
+    }
+    return result;
+})('EventTarget', 'XMLHttpRequestEventTarget');
```

Comment:
This would be clearer if you had a named function called readFixture, and then you could write

const FIXTURES = { EventTarget: readFixture('interface-overview', 'EventTarget'), ...};

I see that other fixtures already exist. Which tests use them and how do those tests read them in? How about adding this readFixture() function to utils.js?

Reply:
As far as I can tell, external fixtures are only used in two tests at the moment:

* [compat.test.js](https://github.com/mdn/kumascript/blob/master/tests/macros/compat.test.js#L13-L20), which tests the Compat.ejs macro
* [dekiscript-page.test.js](https://github.com/mdn/kumascript/blob/master/tests/macros/dekiscript-page.test.js#L46-L53), added by @escattone in https://github.com/mdn/kumascript/pull/240, which I think is testing some Wiki-specific APIs that macros are able to use (specifically [`subpages`](https://github.com/mdn/kumascript/blob/master/src/environment.js#L871), [`subpagesExpand`](https://github.com/mdn/kumascript/blob/master/src/environment.js#L898), and so on.

These utility functions seem like they would be useful, yes. A common fixture for a lot of macros (especially sidebars) is going to be this list of page metadata that's returned by functions like `subPagesExpand` (because sidebars very often want to say "get me all the pages underneath this one" and then do something with that result):

https://github.com/mdn/kumascript/blob/master/macros/WebExtAPISidebar.ejs#L143
https://github.com/mdn/kumascript/blob/master/macros/APIRef.ejs#L44
https://github.com/mdn/kumascript/blob/master/macros/APIRef.ejs#L86
https://github.com/mdn/kumascript/blob/master/macros/ListSubpagesForSidebar.ejs#L29
https://github.com/mdn/kumascript/blob/master/macros/JSRef.ejs#L88

I don't know if there's any scope then for providing a generic utility to mock `subpagesExpand`, or something like that.